x-pack/plugins/observability/server/routes/slo/route.ts

```diff
@@ -225,6 +227,22 @@ const findSLORoute = createObservabilityServerRoute({
   },
 });
 
+const deleteSloInstancesRoute = createObservabilityServerRoute({
+  endpoint: 'POST /api/observability/slos/_delete_instances 2023-10-31',
+  options: {
+    tags: ['access:slo_write'],
```

Comment:
We need to add `access: 'public'` here.

Reply:
last time I added this access public, it kind of removed the `slo_write` permission checks... I need to try again

Reply:
AH there is `tags` param with `access` prefix and `access` param... Ok I'll do this in another PR, as I want to make sure the permission is still working

Reply:
Handled in https://github.com/elastic/kibana/pull/165698